SwiftUIProject/SwiftUIProject/ContentView.swift

```diff
@@ -6,18 +6,99 @@
 
 import SwiftUI
 
+/// Button with Alert and ActionSheet
 struct ContentView: View {
+    
+    private enum Constants {
+        static let exampleAlertText = "Example Alert"
+        static let showButtonText = "Show"
+        static let spacing: CGFloat = 10
+        static let exampleText = "Text!"
+        static let changeColorText = "Change color"
+        static let alertTwoText = "Alert two"
+        static let alertTwoMessageText = "Do you want to change a color?"
+        static let yesText = "Yes"
+        static let exampleActionSheetText = "Example ActionSheet"
+        static let changeWordText = "Change word"
+        static let badWordText = "Death"
+        static let goodWordText = "Life"
+        static let questionWordText = "Which word do yo want to see?"
+    }
+    
     var body: some View {
-        VStack {
-            Image(systemName: "globe")
-                .imageScale(.large)
-                .foregroundColor(.accentColor)
-            Text("Hello, world!")
+        VStack(spacing: Constants.spacing) {
+            taskOne()
+            Text(changeText).foregroundColor(textColor)
+            taskTwo()
+            taskThree()
+            Text(exampleWord)
+            taskFour()
+        }
+    }
+    
+    @State private var isShowAlertOne = false
+    @State private var isShowAlertTwo = false
+    @State private var isActionSheetOne = false
+    @State private var isActionSheetTwo = false
```

Comment:
```suggestion
    @State private var isShowAlertOneShown = false
    @State private var isShowAlertTwoShown = false
    @State private var isActionSheetOneShown = false
    @State private var isActionSheetTwoShown = false
```
   если флаг указывает, что ожидается показ чего-либо, то  на конце пишется Shown